resolwe_bio/management/commands/generate_samples.py

```diff
@@ -0,0 +1,259 @@
+        if organism_name.startswith('Hs'):
+            annotation = {'geo': {'organism': 'Homo sapiens',
+                                  'annotator': random.choice(annotator),
+                                  'source': 'brain',
+                                  'cell_type': random.choice(['neuron', 'glia']),
+                                  'genotype': random.choice(['wildtype', 'CDK6-/-']),
+                                  'experiment_type': 'RNA-Seq',
+                                  'molecule': 'polyA RNA',
+                                  'optional_char': ['Cell line: DBTRG-05MG'],
+                                  'description': "Human brain study. Expression profiling."},
+                          'protocols': {
+                            'fragmentation_method': 'sonication',
+                            'growth_protocol': growth_protocol,
+                            'library_prep': library_prep}}
+        if organism_name.startswith('Mm'):
+            annotation = {'geo': {'organism': 'Mus musculus',
+                                  'annotator': random.choice(annotator),
+                                  'source': 'brain',
+                                  'genotype': random.choice(['wildtype', 'CDK6-/-']),
+                                  'cell_type': random.choice(['neuron', 'microglia']),
+                                  'experiment_type': 'RNA-Seq',
+                                  'molecule': 'polyA RNA',
+                                  'optional_char': ['Cell line: EOC 2'],
+                                  'description': "Mouse brain study. Expression profiling."},
+                          'protocols': {
+                            'fragmentation_method': 'sonication',
+                            'growth_protocol': growth_protocol,
+                            'library_prep': library_prep}}
+        return annotation
+
+    def create_data(self, reads_name='seq_reads', annotated=False):
+        # get test data paths
+        data_path = settings.FLOW_EXECUTOR['DATA_PATH']
+        test_files_path = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', '..', 'tests', 'files'))
+        reads = os.path.join(test_files_path, reads_name + '.fastq.gz')
+        fastqc = os.path.join(test_files_path, reads_name + '_fastqc.zip')
+        bam_mapping = os.path.join(test_files_path, 'alignment_position_sorted.bam')
+        bai = os.path.join(test_files_path, 'alignment_position_sorted.bam.bai')
+        dicty_genes = os.path.join(test_files_path, 'dicty_genes.tab.gz')
+        human_genes = os.path.join(test_files_path, 'human_genes.tab.gz')
+        mouse_genes = os.path.join(test_files_path, 'mouse_genes.tab.gz')
+
+        # crate reads data object
+        d = Data.objects.create(
+            name = self.set_name(),
+            descriptor_schema = DescriptorSchema.objects.get(name='Reads annotation'),
```

Comment:
It is ok, this descriptor schema is written in old format. This means that `label` is used as `name` when registered.